### PR TITLE
BOAS-22: refactor apps/web to accommodate appeals/applications services

### PR DESCRIPTION
**Pull request by brpins**
## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAS-22

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/app/app.router.js [hunk 1]
```diff
@@ -1,54 +1,35 @@
...
-import { registerInspectorLocals } from './inspector/inspector.pipes.js';
-import inspectorRouter from './inspector/inspector.router.js';
-import { registerValidationLocals } from './validation/validation.pipes.js';
-import validationRouter from './validation/validation.router.js';
 
 const router = createRouter();
 
+// In development only, integrate with locally defined user groups
+
+if (config.authDisabled) {
```
Comment (francescobrenci-AR) [inline apps/web/src/server/app/app.router.js:14]: I reckon the check for what Auth to use should be done somewhere else - maybe in `assertIsAuthenticated`

Comment (francescobrenci-AR) [inline apps/web/src/server/app/app.router.js:14]: I reckon the check for what Auth to use should be done somewhere else - maybe in `assertIsAuthenticated`

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

File: apps/web/src/server/app/app.router.js (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/app/app.router.js:14]: ok fair point.
Is `authDisabled = false` always equating to the DEV BTW?

Comment (brpins) [reply apps/web/src/server/app/app.router.js:14]: yeah – it's to allow developers to use the web locally without having to authenticate with azure SSO

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**